pkg/cache/cohort.go

```diff
@@ -85,3 +87,16 @@ func (c *cohort) CCParent() hierarchy.CycleCheckable {
 func (c *cohort) fairWeight() *resource.Quantity {
 	return &c.FairWeight
 }
+
+// Returns all ancestors starting with self and ending with root
+func (c *cohort) PathToRoot() iter.Seq[*cohort] {
```

Comment:
I see there is a small semantic difference between cohort.PathToRoot and cq.PathToRoot - as CQ does not start with self. I'm ok to keep this detail reflected just in comment. 

Just exploring alternative, more explicit, naming options:
- PathFromParentToRoot and PathFromSelfToRoot.
- or just: FromParentToRoot and FromSelfToRoot, assuming the "Path" is implied

Wdyt @KPostOffice @gabesaba ?

Reply:
wdyt about omitting `From`. So, `PathSelfToRoot` and `PathParentToRoot`

Reply:
sgtm, but let's hear from @gabesaba